### PR TITLE
React to rename of "@dotnet/jsinterop" to "@microsoft/dotnet-js-interop"

### PR DESCRIPTION
**Pull request by analogrelay**
This is the reaction to https://github.com/aspnet/Extensions/pull/2634 (when that merges and publishes bits).

~~**DO NOT MERGE** until https://github.com/aspnet/Extensions/pull/2634 is complete. And even then this will probably need to be cherry-picked in to the dependency flow PR.~~

Turns out the dependency flow here is totally manual, so I'll be able to update the URL in this PR once aspnet/Extensions#2634 lands.

### CODE REVIEW

File: src/Components/Web.JS/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "devDependencies": {
     "@aspnet/signalr": "link:../../SignalR/clients/ts/signalr",
     "@aspnet/signalr-protocol-msgpack": "link:../../SignalR/clients/ts/signalr-protocol-msgpack",
-    "@dotnet/jsinterop": "https://dotnet.myget.org/F/aspnetcore-dev/npm/@dotnet/jsinterop/-/@dotnet/jsinterop-3.0.0-preview9.19415.3.tgz",
+    "@microsoft/dotnet-js-interop": "https://dotnet.myget.org/F/aspnetcore-dev/npm/@microsoft/dotnet-js-interop/-/@microsoft/dotnet-js-interop-3.0.0-preview9.19415.3.tgz",
```
Comment (anurse) [inline src/Components/Web.JS/package.json]: What keeps this URL up-to-date with the right version @pranavkm ? I don't want the dependency flows blasting this back to the old name. I didn't see anything else referring to `@dotnet/jsinterop` (I grepped the whole repo) but I may be missing something.

Comment (pranavkm) [reply src/Components/Web.JS/package.json]: It's entirely manual. We update it anytime we add update the JS package which isn't very frequent.

**Review by pranavkm — APPROVED** [review]
Looks good. Thanks!

File: src/Components/Web.JS/package.json (hunk 1 above)
Comment (anurse) [reply src/Components/Web.JS/package.json]: Ok, so we should also update the version number when the extensions PR merges and is built.

**Comment by anurse** [comment]
@pranavkm can you take another quick look? The extensions PR has merged and I've updated the URL.

**Comment by pranavkm** [comment]
CodeCheck doesn't seem to like this change.

**Comment by anurse** [comment]
> CodeCheck doesn't seem to like this change.

I have to do the Blazor JS dance 💃🕺 don't I?

**Comment by anurse** [comment]
Ok, code check should be happy now. Would appreciate a look from any blazor folks who can (@rynowak @SteveSandersonMS @pranavkm @javiercn)

The `@microsoft/dotnet-js-interop` package **is** published to aspnetcore-dev now, so this should be all working. I rebuilt Web.JS and have updated the checked-in version of the script.

**Review by SteveSandersonMS — APPROVED** [review]
The renames look totally reasonable. Assuming the E2E tests are still passing, 👍

**Comment by anurse** [comment]
Oof, I need to get this off my todo list and in the 5.0 branch :). My bad, I'll update and re-spin builds. Should just be a matter of updating the pre-compiled blazor.server.js.

**Comment by pranavkm** [comment]
FYI @SteveSandersonMS \ @javiercn, this change is now in in the master branch.

**Comment by anurse** [comment]
FYI There should be no functional change, it's just aligning with the 5.0 plan to publish the `@dotnet/js-interop` package as `@microsoft/dotnet-js-interop`.

**Pull request merged**